strum_tests/tests/enum_discriminants.rs

```diff
@@ -215,3 +215,45 @@ fn filter_variant_attributes_pass_through() {
         VariantFilterAttrDiscs::from_str("dark_black").unwrap()
     );
 }
+
+#[cfg(bare_pub)]
+#[test]
```

Comment:
One more thing, can you use `cfg_attr` on the `strum_discriminants` attribute to determine between `pub` and `r#pub` instead of duplicating the entire function?

```rust
#[test]
fn override_visibility() {
    mod private {
        // ...

        #[allow(dead_code)]
        #[derive(EnumDiscriminants)]
        #[strum_discriminants(name(PubDiscriminants))]
        #[cfg_attr(bare_pub, strum_discriminants(vis(pub)))]
        #[cfg_attr(not(bare_pub), strum_discriminants(vis(r#pub)))]
        enum PrivateEnum {
            VariantA(bool),
            VariantB(bool),
        }
    }

    // ...
}
```

Reply:
Makes sense, I'll make the change now!

Reply:
Hmmm, looks like my Rust 1.47 doesn't like seeing `r#pub` at all (file is `strum_tests/tests/enum_discriminants.rs`):
```
mark@rche:~/Projects/rust/strum$ cargo test
   Compiling strum_tests v0.19.4 (/home/mark/Projects/rust/strum/strum_tests)
error: unexpected token
   --> strum_tests/tests/enum_discriminants.rs:228:59
    |
228 |         #[cfg_attr(not(bare_pub), strum_discriminants(vis(r#pub)))]
    |                                                           ^^^^^

error[E0433]: failed to resolve: could not find `PubDiscriminants` in `private`
   --> strum_tests/tests/enum_discriminants.rs:236:18
    |
236 |         private::PubDiscriminants::VariantA,
    |                  ^^^^^^^^^^^^^^^^ could not find `PubDiscriminants` in `private`

error[E0433]: failed to resolve: could not find `PubDiscriminants` in `private`
   --> strum_tests/tests/enum_discriminants.rs:237:18
    |
237 |         private::PubDiscriminants::VariantB,
    |                  ^^^^^^^^^^^^^^^^ could not find `PubDiscriminants` in `private`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0433`.
error: could not compile `strum_tests`.
```

Reply:
All right then. Thanks for trying it; let's just leave it as it was :)

Reply:
Sounds good to me, I'll mark this review as resolved.